main.js

```diff
@@ -0,0 +1,25 @@
+var car = document.querySelector('img');
+
+var carDirectionData = {
+  north: 'north',
+  south: 'south',
+  west: 'west',
+  east: 'east'
+}
```

Comment:
this kind of doesnt' really work as a data model
You could have a direction property which stores the direction?
`direction: 'east'`